src/components/navbar/Menu.tsx

```diff
@@ -33,7 +33,7 @@ const Menu = () => {
   const { data: userSession } = useSession();
   const currentUserRole = userSession?.user?.permissions as UserRole;
   const isPermitted = checkPermissionPrivileges(currentUserRole, "evaluator");
-  const isAdmin = checkPermissionPrivileges(currentUserRole, "admin");
+  const isAdmin = currentUserRole === "admin";
```

Comment:
The point of the `useHasPermission` hook is to avoid this type of code

Reply:
Oh ok, I did that because i did not want to repeat each of the permissions, for example check if the user has permission for each of the admin menu
something like `const canResetReviews = useHasPermissions("resetReviews")` for each of those three admin menus

Reply:
But that was the idea behind that hook. If you didn't like the idea we should discuss it, keeping both ways to check for permissions doesn't make sense, it makes the code error-prone.

Reply:
Oh no, I think it's good, I did enjoy using the hook, I just didn't want to have more variables for each menu. it works great as it is

Reply:
Also, what does it mean I don't need to create a new permission, my second goto thought was to create an "accessAdminNav" permissions in the permissions.json.

Reply:
> Also, what does it mean I don't need to create a new permission, my second goto thought was to create an "accessAdminNav" permissions in the permissions.json.

I meant that you already have the `accessReviews` permission that you use for `src/pages/admin/reviews.tsx`, why not use that to display the navigation menu element for that page?

Reply:
Yh I get that but also there is this `isPermitted? (
                      <AdminMenu role={userSession.user?.permissions}>`
 This is meant to determine if an Admin navbar shows. We could use accessReviews because only evaluators and Admins are allowed there, but that isn't good since accessReviews is only meant to give permission to the reviews page.

Reply:
<img width="305" alt="Screenshot 2024-06-18 at 11 45 10" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/90271995/b91f3d3f-7fcb-44b7-9f92-8d5da8dd0dc2">
Talking about this Nav here.